src/common/styles.tsx

```diff
@@ -0,0 +1,66 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+interface AppProps {
+  flexColumnCenter?: boolean;
+}
+
+interface PageTitleProps {
+  mb10?: boolean;
+}
+
+export const GlobalStyles = css`
+  body {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    align-items: center;
+    justify-content: center;
+    background-color: #e5e5e5;
+  }
+
+  input {
+    font-size: 16px;
+  }
+`;
+
+export const PageTitle = styled.h2`
```

Comment:
```suggestion
export const PageTitle = styled.h2<{mb10}: {PageTitleProps}>`
```
아래 사용하고 있는 방법으로 일관성 맞춰주시는게 더 좋을 것 같네요